### DISCUSSION TITLE
GCAM-USA State Applications Community of Practice (slides in replies)

### DISCUSSION THREAD

**Discussion by DLoughlin** [JGCRI/gcam-core#361] [GCAM-USA]
GCAM-USA is increasingly being used by external groups in support of U.S. state level applications. As a way to organize this community and foster collaboration and support, a “GCAM-USA State Applications Community of Practice” is being formed by the GCAM-USA user community.  

Examples of topics to be covered include: 
-	Presenting recent, relevant GLIMPSE and/or GCAM applications
-	Presenting updates to GLIMPSE and the ModelInterface
-	Discussing new GCAM developments that may be of interest to this community, such as disaggregation of the residential sector by income decile
-	Describing approaches for linking GCAM-USA to tools such as COBRA and air quality models
-	Using the CSVtoXML utility to create your own add-on files
-	Setting up and running the GCAM data system
-	Troubleshooting unsolved markets
-	Running GCAM in the cloud
-	Using tools such as PyGCAM to automate sensitivity and uncertainty analysis
-	Wish lists for future features

Meetings are tentatively scheduled for the 4th Friday of each month, 12-1 Eastern. We look forward to interacting with many of you through this Community of Practice. Please email Dan Loughlin (loughlin.dan@epa.gov) for an invitation to the next meeting.

**Comment by DLoughlin** [comment]
Presentation from Feb 22nd: "Maryland's Climate Pathway", presented by Alicia Zhao and Kathleen Kennedy of the Univ. of MD's Center for Global Sustainability: 
[Maryland’s Climate Pathway - GCAM-USA State Application Community Meeting.pdf](https://github.com/JGCRI/gcam-core/files/14909241/Maryland.s.Climate.Pathway.-.GCAM-USA.State.Application.Community.Meeting.pdf)

**Comment by DLoughlin** [comment]
Presentation from Mar 28th: "Substantial air quality and health benefits from combined federal and subnational climate actions in the United States", presented by Xinyuan Huang of Princeton Univ.
[20240328 IRA_Health GCAM-USA-cop.pdf](https://github.com/JGCRI/gcam-core/files/14909317/20240328.IRA_Health.GCAM-USA-cop.pdf)

**Comment by DLoughlin** [comment]
Presentation from April 25th: "Implications of climate change mitigation and socioeconomic development on the U.S. electric power sector", Presented by Lu Liu, an Assistant Professor at Iowa State University. Co-authors were Fatemeh Ganji and Shanna Fellows.
 
[GCAM-USA state application community of practice_Liu.pdf](https://github.com/JGCRI/gcam-core/files/15152380/GCAM-USA.state.application.community.of.practice_Liu.pdf)

**Comment by DLoughlin** [comment]
Presentation on June 27th: "Air quality impacts of decarbonization: An illustrative example." This presentation summarized an application of using GLIMPSE and GCAM-USA to simulate greenhouse gas emission reduction measures, evaluate the associated air pollutant co-reductions, and translate those emission changes into air quality and health benefits using EPA's COBRA model. 
[GCAM-USA_COP_2024.06.27_withDemoSlides.pptx](https://github.com/user-attachments/files/16147743/GCAM-USA_COP_2024.06.27_withDemoSlides.pptx)

**Comment by DLoughlin** [comment]
For the June 27th Community of Practice meeting, Ying Zhang of PNNL presented on GCAM-USA modeling work exploring the net-zero policy impacts on residential energy security within and across income groups.
[GCAM-USA_COP_resid_energy_security_Ying_Zhang.pptx](https://github.com/user-attachments/files/16391825/GCAM-USA_COP_resid_energy_security_Ying_Zhang.pptx)

**Comment by DLoughlin** [comment]
On Sept. 26th, Parisa Javadi (a PhD student at the Univ. of VA) will be made a presentation on "The Impact of Regional Resources and Technology Availability on Carbon Dioxide Removal Potential in the United States". 
[CDR - GCAM.pdf](https://github.com/user-attachments/files/17155974/CDR.-.GCAM.pdf)

**Comment by DLoughlin** [comment]
On Dec. 5th, Di Sheng of PNNL presented on the GCAMUSAJobs package, as well as an application quantifying the job gains associated with a Net Zero scenario.  
[DSheng-GCAMUSAJobs.pdf](https://github.com/user-attachments/files/18030268/DSheng-GCAMUSAJobs.pdf)

In addition, Joyce Kim of EPA's Office of Research and Development demonstrated an application of the Kaya Identify methodology to decompose GCAM-derived, state-level GHG reduction strategies.
[JKim_StateCOP meeting_120524.pdf](https://github.com/user-attachments/files/18030302/JKim_StateCOP.meeting_120524.pdf)

**Comment by DLoughlin** [comment]
For the January 23rd (4-5 EST) meeting of the GCAM-USA State Applications Community of Practice, Anne Claflin and others from the State of Minnesota and the Univ. of MD discussed how they are applying GCAM-USA in the development of their state’s Comprehensive Climate Action Plan (CCAP). Topics covered included stakeholder engagement, scenario development, identification of measures to evaluate, and future modeling directions.
[CGS slides - GCAM-USA CoP 25-1-23.pdf](https://github.com/user-attachments/files/18591472/CGS.slides.-.GCAM-USA.CoP.25-1-23.pdf)

**Comment by steph-vo** [comment]
The Center for Global Sustainability (CGS) at the University of Maryland will continue to host the GCAM-USA State Applications Community of Practice moving forward. Thank you for your patience as we worked with Dan to figure out the logistics of hosting this series. 

With this shift, we would like to take the time to ensure that these convenings fit the needs and desires of this community. We have developed a [Google form](https://docs.google.com/forms/d/e/1FAIpQLScs5fdBncs6v9JgP8kk2FUrGSRrBO3R2vYy_NMiGJlAbvINAQ/viewform?usp=header) to get feedback from you all on (1) the recurring meeting time and (2) topics of interest and your specific GCAM needs. Please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLScs5fdBncs6v9JgP8kk2FUrGSRrBO3R2vYy_NMiGJlAbvINAQ/viewform?usp=header) by April 30, 2025, as we would like to get a meeting on the calendar starting in late May.

In the first meeting back, the state-level program team at CGS will discuss the process they used to modify GCAM-USA for detailed state analysis in Minnesota and Maryland. 

We hope that this community can continue to serve as a place to foster collaboration, learn, and exchange ideas and use cases. Please let me know if you have any questions or concerns (stvo@umd.edu).

**Comment by steph-vo** [comment]
On May 30, 2025, Kathleen Kennedy and Bradley Phelps presented the workflow that they use to adapt GCAM-USA for detailed state-level analysis, using projects in Maryland and Minnesota as examples. Some topics covered include state inventory matching, using state-specific data to adjust GCAM parameters, and how to represent certain policies.
[2025_May_State Modeling Process_CGS.pdf](https://github.com/user-attachments/files/20553596/2025_May_State.Modeling.Process_CGS.pdf)

The community's upcoming meeting on June 27, 2025, will be a discussion on the needs of the community and the balance between GCAM-USA and GLIMPSE. Some questions to think about are:
- What are your needs and how can this group address those needs? 
- What applications are coming from the community utilizing GLIMPSE and GCAM-USA?
- How do we want to balance the emphasis of GLIMPSE and GCAM-USA in this group? How do we prioritize our resources towards each and understand how GCAM-USA feeds into GLIMPSE (and vice versa)?
- How can this community contribute to GLIMPSE development?

**Comment by steph-vo** [comment]
On June 27, 2025, the GCAM-USA State Applications Community of Practice group met to discuss specific needs the community of practice has and the resources it could help provide. There were several ideas brought up, and the high-level takeaways are summarized below. We will be using these ideas to help direct short- and long-term meeting topics as well as future initiatives for this group. 

- **GLIMPSE developments**: Participants were interested in the future development of GLIMPSE (e.g., the integration of GCAM 8.2). For questions or inquiries about GLIMPSE development, please contact Jeff Cole at EPA ([cole.jefferson@epa.gov](mailto:cole.jefferson@epa.gov)). This group can be a place to help participants keep up-to-date about GLIMPSE developments.

- **GCAM-USA developments**: Even with the recent GCAM version update, GCAM-USA does not have many large developments, such as detailed/disaggregated industry. This group can also be a place to help participants keep up-to-date about GCAM-USA developments.

- **Trainings and resources**: Participants requested better training resources for getting started using GCAM-USA, especially after participating in GLIMPSE trainings hosted by EPA. There is a need for peer-to-peer support in understanding GCAM-USA, and one idea would be to create a standardized repository of input files for different groups to use. 

- **Areas of application**: Participants also indicated that it could be beneficial for the group to better understand the distinction between GLIMPSE and GCAM-USA, and in which applications to use each tool. 

- **Topics of interest**: Some expressed interest in learning more about how GCAM-USA can be used to support different needs, such as research on energy demand from data centers or state-level emission reduction policies.

**Comment by steph-vo** [comment]
On July 25, 2025, Dr. Rachel Hoesly presented about detailed industry developments in GCAM-USA. I've pasted links to her slides and a recording of the meeting below.

- Meeting recording: https://drive.google.com/file/d/1zj60alcn1__CnHL3UcFPh5q_xe-taPo5/view?usp=sharing 
- Detailed industry presentation: https://drive.google.com/file/d/1LfOVAl10J0nYZe23oyzn1O4Y632vbnRP/view?usp=sharing

**Comment by DLoughlin** [comment]
On August 29th, 2025, Huilin Luo (Ph.D. candidate) of Princeton University made a presentation on "Modeling the impacts of policy sequencing on energy decarbonization."

* Meeting recording: https://drive.google.com/file/d/1Kkmxl7J9-Qi_2JiTjQ-R9fgEZ5gyVSyp/view?usp=sharing 
* Presentation: https://drive.google.com/file/d/1cF4BqZORwsJd65P9Gaa2dYh_uhJjNiIl/view?usp=sharing

**Comment by DLoughlin** [comment]
On September 26th, Dr. Joyce Kim of the U.S. EPA made a presentation entitled "GitHub repository release for Kaya decomposition factor analysis with GCAM/GLIMPSE modeling output data."

* Meeting recording: https://drive.google.com/file/d/12O8KHM33r_-B7-riTAsWlbkD6xiHqSNi/view?usp=sharing
* Presentation: https://drive.google.com/file/d/1fVUw6dOh8tzXM-bY0lOlIvU8uz_ARe6G/view?usp=sharing

**Comment by DLoughlin** [comment]
On October 31st, Jinyu Shiwang (Ph.D. candidate) of Princeton University made a presentation entitled: "Technology indicators enable early detection of decarbonization failures." 

* Meeting recording: https://drive.google.com/file/d/1yV6J7cvSlMkTL6h0EOOGp4aC2XwhT1Pa/view?usp=sharing
* Presentation: https://drive.google.com/file/d/1fh2SpZ_0q4sAjgXaO0oc5QXo3xi27Zbj/view?usp=sharing

**Comment by DLoughlin** [comment]
On November 21st, Dan Loughlin made a presentation entitled: "Evaluating air quality-related health impacts with GCAM-USA." 

* Meeting recording: https://drive.google.com/file/d/1c_L1r_9x-Ses-GW_ihUloMQGxhXgn1TN/view?usp=sharing
* Presentation: https://drive.google.com/file/d/1UprMTuEYcVwr56VhDZqh5Sg5q33elvl0/view?usp=sharing

**Comment by steph-vo** [comment]
On January 31, 2026, Kathleen Kennedy and Bradley Phelps presented to the group about work that the Center for Global Sustainability is doing with the State of Maryland to update modeling for the State's climate goals. 

- Recording: https://drive.google.com/file/d/10UbV6uan_ZZL-CNz49bSD_TKu06gdCv2/view?usp=sharing 
- Slides: https://drive.google.com/file/d/1fWUqwMzrG9ZaTJYjzLNkA2yLUrtQ4DHd/view?usp=sharing

**Comment by steph-vo** [comment]
On February 27, 2026, Dan Loughlin gave a short overview of the new GLIMPSE ModelInterface at the beginning of the meeting. Naushin Tabassum presented her work on evaluating the air quality and health co-benefits of expanding the Regional Greenhouse Gas Initiative to Pennsylvania and Virginia. And, Kiara Ordonez Olazabal presented her work on developing a state tiering system to represent heterogeneous ambition across U.S. states to quantify the implications of expanded state-level climate leadership.

You can find the recording and slides below:

- Recording: https://drive.google.com/file/d/1TwBqOL7ewgWOPQXOeBMTUV_m1_gUPnKH/view?usp=drive_link

- Nashin's slides: https://drive.google.com/file/d/1h396OEVBHwgrA4Ch7fbO57Gut3HKNaHM/view?usp=drive_link

- Kiara's slides: https://drive.google.com/file/d/1ro3TP5FkQk-30F8vEBBUafEzXNlXQvYV/view?usp=drive_link

**Comment by steph-vo** [comment]
On March 27, 2026, Candelaria Bergero joined us to present her work examining how residual emissions may perpetuate community-scale inequalities of U.S. air pollution in net-zero scenarios. Please find the meeting recording, Candelaria's slides, and a link to the associated preprint below. 

- Recording: https://drive.google.com/file/d/1IBNbXWdt1Fga9PC0cUvHIEjm_ZjCUFmr/view?usp=sharing
- Slides: https://drive.google.com/file/d/1qnKFqXzY6fDWksCmJ2uAvjBICSp47uXV/view?usp=sharing
- Preprint link: https://www.researchsquare.com/article/rs-7359464/v1

Additionally, a reminder that GLIMPSE-CE 2.0 has been released! This release includes GCAM/GCAM-USA model version 8.2, as well as improvements to ScenarioBuilder and ModelInterface. You can find more information at the link below. Dan Loughlin will be hosting a training series for this in the near future - email Dan at danloughlin@gmail.com to sign up for updates. 

- https://github.com/DLoughlin/GLIMPSE-CE/releases/tag/v2.0-2026.03.05

**Comment by DLoughlin** [comment]
Matthew Binsted, a scientist at PNNL's Joint Global Change Research Institute, joined us on July 31 to present an overview of the GCAM-USA-dispatch model. Slides and presentation recording are linked below:

Slides: https://drive.google.com/file/d/1U4SpY4sMBzlrdE7SLMnxFFBtC4v1G55Z/view?usp=sharing 
Recording: https://drive.google.com/file/d/1etLyv7bp_95Wah2GSSYQ70rhRmsnPbvC/view?usp=sharing

Title:  GCAM-USA Subseasonal Electric Dispatch and Storage

Description:  The electric power sector is expected to grow in size, importance, and complexity around the world as economies expand and electric supply technologies and demand patterns evolve in significant ways. Newer, rapidly growing sectors could alter the temporal profile of electricity demand from historical patterns. In addition, substantial increases in variable renewable energy technologies could occur, adding variability and uncertainty to the diurnal and seasonal profile of electric supply. In this context, the emergence of modular, flexible electricity storage technologies may have profound impacts on the structure and operation of electric power systems. This presentation describes a new approach for incorporating grid-based electricity storage investment and operation into the electric sector dynamics of the Global Change Analysis Model-USA (GCAM-USA) – addressing a gap in multi-sector dynamics models.

**Comment by DLoughlin** [comment]
On Aug. 28th, 2026, Dan Loughlin presented "How GCAM works - Markets, logits, share weights, and calibration"

Slides:  [HowGCAMWorks-2026.08.30.pptx](https://github.com/user-attachments/files/31619056/HowGCAMWorks-2026.08.30.pptx)
Recording: [Link](https://drive.google.com/file/d/1wSp7YeA2Z1b74AuyD8DuKpGsbSdEv7zA/view?usp=sharing)

Description: GCAM's simulation process involves stepping through time, and in each step finding the set of prices at which supply equals demands across thousands of markets simultaneously. In doing so, GCAM simulates the market shares for technologies using a logit equation. Important parameters in this equation include the levelized cost of each technology competing within the market, technology-specific share weights that represent bias and unmodeled issues, and a market-specific logit exponent that captures sensitivity to cost differences. Policies also typically operate through the logit function by impacting technology costs through taxes or subsidies. Understanding how the logit equation works, as well as its strengths and limitations, are critical skills for GCAM modelers. In this presentation, Dan Loughlin will describe market operation in GCAM and provide an example related to passenger car electrification.
